utils/_context/_scenarios.py

```diff
@@ -972,6 +1014,13 @@ class scenarios:
         doc="Spawns tracer, agent, and a full set of database. Test the intgrations of thoise database with tracers",
     )
 
+    crossed_integrations = EndToEndScenario(
+        "CROSSED_INTEGRATIONS",
```

Comment:
`CROSSED_INTEGRATIONS` makes me think that we're testing from one integration (ie: a Kafka library) against another integration (ie: another Kafka library or something else) within one programming language but this test would allow someone to test across programming languages. 

How about including `programming language` or something to that effect that implies we're crossing the programming languages as part of this test?

Suggestions/examples:
- `CROSSED_LANG_INTEGRATIONS`
- `CROSSED_TRACING_LIBRARY`
- `CROSSED_TRACING_LANG`

What do you think @tabgok and @cbeauchesne ?

Reply:
My personal opinion goes for `CROSSED_TRACING_LIBRARY`, but I would use `CROSSED_TRACING_LIBRARIES`, to make clear that several libs are involved